js/src/ui/TxList/txList.css

```diff
@@ -42,10 +42,11 @@
     }
 
     &.timestamp {
-      padding-top: 1.5em;
-      text-align: right;
+      max-width: 5em;
+      padding-top: 0.75em;
+      text-align: center;
       line-height: 1.5em;
-      opacity: 0.5;
+      color: grey;
```

Comment:
All links and others (sumbitting/ cancel/ edit ...) will be 50% as well. Which is a problem to see/links look strange. I don't mind adding more css, so that the main is 0.5 opacity. But I can't keep the css the way it is. Suggestions are welcome.